Automattic/pocket-casts-ios
# Update bookmarks icon in settings

| 📘 Part of: # |  
|:---:|

Updates the bookmarks icon on the profile screen for the correct version.
| Before | After |
| - | - |
| ![Simulator Screenshot - iPhone 15 - 2024-08-09 at 23 24 05](https://github.com/user-attachments/assets/99db9104-2967-4aea-91b0-6a8757b58760) |  ![Simulator Screenshot - iPhone 15 - 2024-08-09 at 23 34 09](https://github.com/user-attachments/assets/30f11ba0-1281-4905-b5e3-df9f6f588292) |


## To test

- Start the app
- Go to Profile
- Check the Bookmarks icon is correct

## Checklist
